packages/plugin-loyalties-api/src/models/ScoreCampaigns.ts

```
@@ -475,7 +475,7 @@
         );
       }
 
-      this.updateOwnerScore({ ownerId, ownerType, updatedCustomFieldsData });
+      await this.updateOwnerScore({ ownerId, ownerType, updatedCustomFieldsData });
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)